docs/HighLevelSILOptimizations.rst

```diff
@@ -117,15 +117,13 @@ Cloning code from the standard library
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The Swift compiler can copy code from the standard library into the
-application. This allows the optimizer to inline calls from stdlib and improve
-the performance of code that uses common operators such as '++' or basic
-containers such as Array. However, importing code from the standard library can
-increase the binary size. Marking functions with @_semantics("stdlib_binary_only")
-will prevent the copying of the marked function from the standard library into the
-user program.
-
-Notice that this annotation is similar to the resilient annotation that will
-disallow the cloning of code into the user application.
+application for functions marked @_inlinable. This allows the optimizer to
+inline calls from the stdlib and improve the performance of code that uses
+common operators such as '++' or basic containers such as Array. However,
```

Comment:
I know it's not your words, but you touched it, so... `++` does not exit any more.